[ES] Add `Client/Options/Offset_Wizard` and `Client/Options/Ignore_list` (#9503)
ppy/osu-wiki: wiki/Client/Options/Offset_Wizard/es.md

# Asistente de compensación

> *Véase también: [Cómo usar el asistente de compensación](/wiki/Guides/How_to_Use_the_Offset_Wizard) y [Compensación universal](/wiki/Offset/Universal_offset)*

Reviewer: ```suggestion
*Véase también: [Cómo usar el asistente de compensación](/wiki/Guides/How_to_use_the_Offset_Wizard) y [Compensación universal](/wiki/Offset/Universal_offset)*
```

> El **Asistente de compensación** es una herramienta integrada que ayuda al usuario a calibrar la [compensación universal](/wiki/Offset/Universal_offset) al valor correcto para su dispositivo. Esto es necesario cuando los [hitsounds](/wiki/Beatmapping/Hitsound) no están sincronizados con la música, en cada mapa que juega. Para obtener más información sobre cómo hacer esto, consulte [Cómo usar el asistente de compensación](/wiki/Guides/How_to_Use_the_Offset_Wizard).

Reviewer: ```suggestion
El **asistente de compensación** es una herramienta integrada que ayuda al usuario a calibrar la [compensación universal](/wiki/Offset/Universal_offset) al valor correcto para su dispositivo. Esto es necesario cuando los [hitsounds](/wiki/Beatmapping/Hitsound) no están sincronizados con la música, en cada beatmap que juega. Para obtener más información sobre cómo hacer esto, véase [Cómo usar el asistente de compensación](/wiki/Guides/How_to_Use_the_Offset_Wizard).
```

> El **asistente de compensación** es una herramienta integrada que ayuda al usuario a calibrar la [compensación universal](/wiki/Offset/Universal_offset) al valor correcto para su dispositivo. Esto es necesario cuando los [hitsounds](/wiki/Beatmapping/Hitsound) no están sincronizados con la música, en cada beatmap que juega. Para obtener más información sobre cómo hacer esto, véase [Cómo usar el asistente de compensación](/wiki/Guides/How_to_Use_the_Offset_Wizard).

Reviewer: ```suggestion
El **asistente de compensación** es una herramienta integrada que ayuda al usuario a calibrar la [compensación universal](/wiki/Offset/Universal_offset) al valor correcto para su dispositivo. Esto es necesario cuando los [hitsounds](/wiki/Beatmapping/Hitsound) no están sincronizados con la música, en cada beatmap que juega. Para obtener más información sobre cómo hacer esto, véase [Cómo usar el asistente de compensación](/wiki/Guides/How_to_use_the_Offset_Wizard).
```

> Para acceder al asistente de compensación, abra el menú de [Opciones](/wiki/Client/Options) y escriba `compensación` en la búsqueda rápida. A continuación, haga clic en el botón `Compensación de sonido global`.

Reviewer: ```suggestion
Para acceder al asistente de compensación, abra el menú de [opciones](/wiki/Client/Options) y escriba `compensación` en la búsqueda rápida. A continuación, haga clic en el botón `Compensación de sonido global`.
```

> Al abrir el asistente de compensación, el usuario recibe el tictac de un metrónomo y barras verdes verticales que se desplazan repetidamente por la pantalla. La canción que se seleccionó por última vez comenzará a reproducirse, aunque esto se puede cambiar a través de la rocola en la esquina superior derecha de la pantalla.

Reviewer: ```suggestion
Al abrir el asistente de compensación, el usuario recibe el tictac de un metrónomo y barras verdes verticales que se desplazan repetidamente por la pantalla. La canción que se seleccionó por última vez comenzará a reproducirse, aunque esto se puede cambiar a través de la gramola en la esquina superior derecha de la pantalla.
```

Author: Creo que es mejor "rocola". Su función para elegir canciones es similar a esa sección del juego.

Reviewer: maquina de discos

Author: Mmm, no sé... No es mala, pero veo esa parte más como una rocola, ya que te dan una lista y tienes la posibilidad de buscar canciones por su nombre. 

Podría ser lo mismo con los discos, pero estos tienes que ponerlos manualmente y no le veo mucha similitud.

Author: Raro, estoy seguro que en la gran mayoría de países de Latinoamérica saben que es una rocola, aunque pueda que me equivoque.

Como mencioné, no me parecía mala y solo lo hacía por la similitud. Así que supongo que está bien aceptar el cambio.

La compensación universal actual se mostrará en la esquina superior izquierda de la pantalla. Se puede cambiar presionando `Flecha arriba` y `Flecha abajo`, o usando la rueda de desplazamiento del ratón.

> Debajo de la pantalla del valor de la compensación universal hay tres opciones que cambian la velocidad del metrónomo. Habilitar `Medio tic` duplicará la cantidad de tiempo entre tics en el metrónomo, haciéndolo más lento. Esto también mostrará la opción `Hacer tic en otro ritmo`. Habilitar `Hacer tic en otro ritmo` mantendrá el tiempo entre los tics del metrónomo igual que en `Medio tic`, pero en un ritmo diferente. Por último, habilitar el `Doble tic` reducirá a la mitad la cantidad de tiempo entre tics en el metrónomo, haciéndolo más rápido. Estas opciones se utilizan para ajustar el metrónomo, para que se adapte mejor a la canción que se está reproduciendo.

Reviewer: ```suggestion
Debajo de la pantalla del valor de la compensación universal hay tres opciones que cambian la velocidad del metrónomo. Habilitar `Medio tic` duplicará la cantidad de tiempo entre tics en el metrónomo, haciéndolo más lento. Esto también mostrará la opción `Hacer tic en otro ritmo`. Habilitar `Hacer tic en otro ritmo` mantendrá el tiempo entre los tics del metrónomo igual que en `Medio tic`, pero en un ritmo diferente. Por último, habilitar `Doble tic` reducirá a la mitad la cantidad de tiempo entre tics en el metrónomo, haciéndolo más rápido. Estas opciones se utilizan para ajustar el metrónomo, para que se adapte mejor a la canción que se está reproduciendo.
```

> Las barras verdes que se desplazan por la pantalla son "tics". Cuando pasan a través de la línea vertical en el centro de la pantalla, parpadearán en blanco y reproducirán un sonido de tic para representar el ritmo. Los tics agregan un aspecto visual útil en el proceso de calibración de la compensación universal.

Reviewer: ```suggestion
Las barras verdes que se desplazan por la pantalla son «tics». Cuando pasan a través de la línea vertical en el centro de la pantalla, parpadearán en blanco y reproducirán un sonido de tic para representar el ritmo. Los tics agregan un aspecto visual útil en el proceso de calibración de la compensación universal.
```
